macros/get_credit_card_payment_txn_columns.sql

```diff
@@ -0,0 +1,20 @@
+{% macro get_credit_card_payment_txn_columns() %}
+
+{% set columns = [    	
+    {"name": "id", "datatype": dbt_utils.type_string()},
+    {"name": "_fivetran_deleted", "datatype": "boolean"},
+    {"name": "_fivetran_synced", "datatype": dbt_utils.type_timestamp()},
+    {"name": "amount", "datatype": dbt_utils.type_float()},
+    {"name": "bank_account_id", "datatype": dbt_utils.type_string()},
+    {"name": "created_at", "datatype": dbt_utils.type_timestamp()},
+    {"name": "credit_card_account_id", "datatype": dbt_utils.type_string()},
+    {"name": "currency_id", "datatype": dbt_utils.type_string()},
+    {"name": "sync_token", "datatype": dbt_utils.type_string()},
+    {"name": "transaction_date", "datatype": "date"},
```

Comment:
is there a reason where `datatype` here is `date` rather than `dbt_utils.type_timestamp()`?